support/assetvalidation/.gitignore

```diff
@@ -0,0 +1,166 @@
+# Custom
```

Comment:
There's a lot in the gitignore here.  Would suggest to remove all of it and only add the entries that are actually needed.  Otherwise someone might trip on this later when a file is ignored on error